addons/stock/report/stock_forecasted.py

```diff
@@ -192,7 +192,10 @@ def _get_report_moves_fields(self):
 
     def _get_report_lines(self, product_template_ids, product_ids, wh_location_ids, wh_stock_location, read=True):
 
-        def _get_out_move_reserved_data(out, linked_moves, used_reserved_moves, currents):
+        def _get_out_move_reserved_data(out, linked_moves, used_reserved_moves, currents, wh_stock_location=wh_stock_location):
```

Comment:
```suggestion
        def _get_out_move_reserved_data(out, linked_moves, used_reserved_moves, currents, wh_stock_location):
```
Seems kinda dubious, and I don't see any benefits, as it's only called at one place anyway. :slightly_smiling_face: